### DISCUSSION TITLE
Tried to import Android SDK but get 2 errors "groovy.lang.MissingPropertyException: Cannot get property 'signing.keyId' on extra properties extension as it does not exist"

### DISCUSSION THREAD

**Discussion by sanazaheer1400** [ant-media/Ant-Media-Server#5684] [General]
I follow the tutorial of [https://antmedia.io/docs/guides/developer-sdk-and-api/sdk-integration/android-sdk/](url) to use ant media server in android application. But when I import the module in the application it gives me 2 errors "groovy.lang.MissingPropertyException: Cannot get property 'signing.keyId' on extra properties extension as it does not exist"
"groovy.lang.MissingPropertyException: Could not get unknown property 'release' for SoftwareComponentInternal set of type org.gradle.api.internal.component.DefaultSoftwareComponentContainer." How to resolve these issues. 

Operating system and version: Mac OS version 14.0
Java version: 17.0.6 Android Studio Giraffe

**Comment by tremzo** [comment]
@sanazaheer1400 what happens if you just download the SDK from here https://github.com/ant-media/WebRTC-Android-SDK and then try to run the sample app in a emulator? Do you still get any errors?

<img width="1687" alt="Screenshot 2023-10-13 at 11 09 03" src="https://github.com/ant-media/Ant-Media-Server/assets/121940452/8b8d78ce-0600-43d8-8a64-b78826f25cfa">

**Reply by sanazaheer1400** [reply]
No it's working fine. Why it gives error when start with the new project?

**Reply by lastpeony** [reply]
Did you get this error when you try to import the sdk as a module?
https://antmedia.io/docs/guides/developer-sdk-and-api/sdk-integration/android-sdk/#import-webrtc-sdk-as-a-module-to-android-project
